src/components/Portrait/style.module.scss

```diff
@@ -0,0 +1,11 @@
+@use '../../themes/theme.scss';
```

Comment:
we can import also import `@use '../../themes/theme.scss'; as *`. 

as result we don't have to write `theme` all of the time but just
`  box-shadow: $red-box-shadow;`

since we do not use any external styling modules I think that might be a good convention to use

Reply:
I fully agree